x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/installation_stats_collector.ts

```diff
@@ -100,10 +100,10 @@ const getInstalledPackagePolicies = (
   const installationStats = packagePolicies.flatMap(
     (packagePolicy: PackagePolicy): CloudSecurityInstallationStats[] =>
       packagePolicy.policy_ids.map((agentPolicyId) => {
-        const agentCounts =
-          agentPolicies?.find((agentPolicy) => agentPolicy?.id === agentPolicyId)?.agents ?? 0;
+        const agentPolicy = agentPolicies?.find((agentPolicy) => agentPolicy?.id === agentPolicyId);
 
-        const isAgentless = agentPolicyId === AGENTLESS_POLICY_ID;
+        const agentCounts = agentPolicy?.agents ?? 0;
+        const isAgentless = agentPolicy?.supports_agentless ?? false;
```

Comment:
using double ternirary will cause falsy values to be used as true, for example if `agentPolicy?.supports_agentless` is equal 0, `isAgentless` will be 0.
If that's what you want its fine but its worth to take note. if you just want a boolean value it makes sense to just use
`!!agentPolicy?.supports_agentless`

Reply:
why would  `agentPolicy?.support_agentless` equal to 0?

Reply:
no reason. but using `??` suggest you **expect** a falsy value and you are fine it unless its `undefined` or `null`. if you dont expect it, why use `??`?

Reply:
nice, thanks.